ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -290,6 +290,15 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: vsphere-upi-f14
```

Comment:
personally I think this job could be renamed with 5-masters keyword to indicate what it's testing, and also f14 sounds like a bit frequent.

Reply:
agree.